examples/stable-diffusion/README.md

```diff
@@ -457,6 +457,51 @@ python image_to_video_generation.py \
     --bf16
 ```
 
+### Image-to-Video with Wan 2.2
+Wan2.2 is a comprehensive and open suite of video foundation models. Please refer to [Huggingface Wan2.2 doc](https://huggingface.co/Wan-AI/Wan2.2-TI2V-5B)
+
+Here is how to generate a video with one image and text prompt:
+
+```bash
+PT_HPU_LAZY_MODE=1 \
+python image_to_video_generation.py \
+    --model_name_or_path "Wan-AI/Wan2.2-TI2V-5B-Diffusers" \
+    --image_path "https://raw.githubusercontent.com/Wan-Video/Wan2.2/main/examples/i2v_input.JPG" \
+    --video_save_dir ./wan2.2-output \
+    --prompts "The cat removes the glasses from its eyes." \
+    --use_habana \
+    --use_hpu_graphs \
+    --height 1088 \
+    --width 800 \
+    --fps 24 \
+    --num_frames 121 \
+    --sdp_on_bf16 \
+    --bf16 
```

Comment:
It seems running this example requires `ftfy` to be installed. Can you add it to the requirements please?